cmd/composectl/cmd/ps.go

```diff
@@ -197,15 +219,15 @@ func printAppStatuses(appStatuses map[string]*App, format string) {
 	}
 }
 
-func getAppUri(storeApps []*compose.AppRef, appName string) string {
-	var foundApp string
+func checkAppInStore(storeApps []*compose.AppRef, appName string) []*compose.AppRef {
+	var foundApps []*compose.AppRef
 	for _, a := range storeApps {
 		if a.Name == appName {
 			// TODO: handle the case when there are more than two one version of the same app in the store
```

Comment:
is there anything we should still add to better handle multiple versions, or this `TODO` comment can be removed?

Reply:
Yes, there are rooms for improvement. We've never supported multiple app versions in the "ps" command (not in "ps <uri>"). This PR change mimics the logic we used to have in aklite, in the aklite's restorableappengine. So, I would leave to do.

It is not easy or even impossible to implement this improvement right now because there is no proper "mapping" between app content in the store (reset-apps) and in the project/install directory (compose-apps). This is a quite big story I've been contemplating and considering for the future roadmap...